google_compute_engine/metadata_scripts/script_retriever.py

```diff
@@ -50,6 +50,18 @@ def _DownloadGsUrl(self, url, dest_dir):
     Returns:
       string, the path to the file storing the metadata script.
     """
+    try:
+       subprocess.check_call(
+         ['which', 'gsutil'],
+         stdout=subprocess.PIPE,
+         stderr=subprocess.PIPE
+       )
+    except subprocess.CalledProcessError:
+      self.logger.warning(
+        'gsutil is not installed, cannot download items from Google Storage'
+      )
```

Comment:
Please also update the failing tests and make sure this code block is fully tested.